### ISSUE TITLE
constructor and destructor definitions have types specified

### ISSUE THREAD

**Issue by goatshriek** [goatshriek/wrapture#72]
In the latest version (0.4.0) definitions generated for class constructors and destructors have a type of 'void', which causes compilation problems. Some example generated code:

```cpp
void Element::Element( const char *name ) {
  this->equivalent = stumpless_new_element( name );
}

void Element::~Element( void ) {
  stumpless_destroy_element( this->equivalent );
}
```

Which resulted from the following spec:

```yaml
constructors:
  - wrapped-function:
      name: "stumpless_new_element"
      includes:
        - "stumpless/entry.h"
      params:
        - name: "name"
          type: "const char *"
      return:
        type: "equivalent-struct-pointer"
destructor:
  wrapped-function:
    name: "stumpless_destroy_element"
    includes:
      - "stumpless/entry.h"
    params:
      - name: "equivalent-struct-pointer"
```

**Comment by goatshriek** [comment]
Fix will be included in pending patch release 0.4.1.

**Comment by goatshriek** [comment]
resolved in #73 which is included in release v0.4.1
